pkg/rid/store/cockroach/identification_service_area.go

```diff
@@ -21,14 +21,14 @@ const (
 	updateISAFields = "id, url, cells, starts_at, ends_at, updated_at"
 )
 
-// ISAStore is an implementation of the ISARepo for CRDB.
-type ISAStore struct {
+// isa is an implementation of the ISARepo for CRDB.
+type isa struct {
```

Comment:
Could we keep isaRepo here?  That seems like a more descriptive data type than "ISA" which is an entirely different data type.